azure-maven-plugin-lib/src/main/resources/schema/com/microsoft/azure/toolkit/maven/asa/Cluster.json

```diff
@@ -0,0 +1,22 @@
+{
+	"$schema": "http://json-schema.org/draft-07/schema#",
```

Comment:
Please unify the indentation style and align with existing schema indentations. I see 2 spaces, 4 spaces, 8 spaces in this PR.